petl/transform/conversions.py

```diff
@@ -297,7 +298,7 @@ def __init__(self, source, converters=None, failonerror=False,
             self.converters = dict([(i, v) for i, v in enumerate(converters)])
         else:
             raise ArgumentError('unexpected converters: %r' % converters)
-        self.failonerror = failonerror
+        self.failonerror = failonerror or config.failonerror
```

Comment:
Not sure this is right. Shouldn't the user-provided value override the config? I.e., maybe the default keyword argument should be ``failonerror=None`` then you could have:

```
if failonerror is None:
    self.failonerror = config.failonerror
else:
    self.failonerror = failonerror
```

?